soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -50,18 +54,51 @@ def send_cmd(self, cmd):
 
     def download(self, filename):
```

Comment:
I know the ship has sailed, but isn't this... "uploading" the file?

Reply:
En... you are right, The client side uploads and the server side downloads the FW.  Fixed it.